contrib/rpm/bitcoin-0.12.0-libressl.patch

```diff
@@ -0,0 +1,24 @@
+diff -ur bitcoin-0.12.0.orig/src/init.cpp bitcoin-0.12.0/src/init.cpp
+--- bitcoin-0.12.0.orig/src/init.cpp	2015-12-31 16:00:00.000000000 -0800
++++ bitcoin-0.12.0/src/init.cpp	2016-02-23 06:03:47.133227757 -0800
+@@ -1075,7 +1075,7 @@
+     if (fPrintToDebugLog)
+         OpenDebugLog();
+ 
+-#if (OPENSSL_VERSION_NUMBER < 0x10100000L)
++#if defined(LIBRESSL_VERSION_NUMBER) || (OPENSSL_VERSION_NUMBER < 0x10100000L)
```

Comment:
Correct but the patch is needed for 0.12.0 source tarball as distributed by bitcoin.org. The README.md file states that the patch will likely not be needed when 0.12.1 is released.